tor/core/admin_commands.py

```diff
@@ -15,16 +15,16 @@
 def process_command(reply, config):
     """
     This function processes any commands send to the bot via PM with a subject
-    that stars with a !. The basic flow is read JSON file, look for key with
-    same subject, check if the caller is mod, or is in the list of allowed
```

Comment:
The comma on this line is a parenthetical comma that pairs with the one on line 20. Feel free to switch these out for parentheses if you want.